commons/com.b2international.index/.classpath

```diff
@@ -7,31 +7,31 @@
 	</classpathentry>
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
 	<classpathentry kind="src" path="src/"/>
-	<classpathentry exported="true" kind="lib" path="lib/aggs-matrix-stats-client-7.9.1.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/analysis-common-7.9.1.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/aggs-matrix-stats-client-7.9.2.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/analysis-common-7.9.2.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/antlr4-runtime-4.5.3.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/asm-7.2.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/asm-analysis-7.2.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/asm-commons-7.2.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/asm-tree-7.2.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/asm-util-7.2.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/compiler-0.9.6.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/elasticsearch-7.9.1.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/elasticsearch-cli-7.9.1.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/elasticsearch-core-7.9.1.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/elasticsearch-geo-7.9.1.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/elasticsearch-rest-client-7.9.1.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/elasticsearch-rest-high-level-client-7.9.1.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/elasticsearch-secure-sm-7.9.1.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/elasticsearch-ssl-config-7.9.1.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/elasticsearch-x-content-7.9.1.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/elasticsearch-7.9.2.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/elasticsearch-cli-7.9.2.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/elasticsearch-core-7.9.2.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/elasticsearch-geo-7.9.2.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/elasticsearch-rest-client-7.9.2.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/elasticsearch-rest-high-level-client-7.9.2.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/elasticsearch-secure-sm-7.9.2.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/elasticsearch-ssl-config-7.9.2.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/elasticsearch-x-content-7.9.2.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/hppc-0.8.1.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/hppcrt-0.7.5.jar"/>
```

Comment:
hppcrt is no longer included in the Windows .zip installer, we might still need it for something.

Reply:
I'll remove it, we are not exposing that embedded dependency to other modules, so we don't need it.